### ISSUE TITLE
NTR: Peroxidase stain low or absent in neutrophils and monocytes (Lab: blood chemistry)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8353]
New term request Peroxidase stain low or absent in neutrophils and monocytes (elthat_130621134034):

**Comment by pnrobinson** [comment]
@MickeySegal
is this for Myeloperoxidase deficiency ? 
I am not sure this is still a clinical assay?

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000513
- label: Diminished neutrophil myeloperoxidase activity
- parents: [Abnormal neutrophil morphology (HP:0011992)](https://hpo.jax.org/app/browse/term/HP:0011992)
- pmid: PMID:9354683
- def: Activity of the enzyme myeloperoxidase (EC 1.11.1.7) in neutrophils below the lower limit of normal. This feature can be measured by peroxidase cytochemistry or biochemical assays. Myeloperoxidase is a lysosomal hemoprotein located in the azurophilic granules of polymorphonuclear leukocytes and monocytes.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8353
- synonyms:
